helm/chaos-mesh/values.yaml

```diff
@@ -139,3 +139,10 @@ preJobs:
 
 postJobs:
   podAnnotations: {}
+
+bpfki:
+  image: pingcap/chaos-kernel:latest
+  imagePullPolicy: Always
+  grpcPort: 50051
+
+  podAnnotations: {}
```

Comment:
Seems bpfki is deployed in the same pod with chaos-daemon, so what about moving this part to be part of `chaos-daemon` in `values.yaml`? What do you think?

And I am not that familar with bpfki, can I ask is there any case that we don't want to deploy bpfki, only deploy chaos-daemon?  If that, I think it is better to add an `enable` option

Reply:
bpfki have performance issues on every container(not only the inject one)